fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FetchResourceChangesDAO.java

```diff
@@ -84,31 +114,53 @@ public FetchResourceChangesDAO(IDatabaseTranslator tx, String schemaName, int re
         }
 
         if (fromTstamp != null) {
-            query.append(" AND c.change_tstamp >= ? ");
+            if (historySortOrder == HistorySortOrder.ASC_LAST_UPDATED) {
+                query.append(" AND c.change_tstamp >= ? ");
+            } else {
+                query.append(" AND c.change_tstamp <= ? ");
```

Comment:
does this mean we basically flip the meaning of `_since` if the sort order is DESC_LAST_UPDATED?

Reply:
at the moment, yes.

Reply:
changed the semantics. See Conformance.md